MapboxNavigation/NavigationMapView.swift

```diff
@@ -120,6 +120,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     @objc dynamic public var buildingDefaultColor: UIColor = .defaultBuildingColor
     @objc dynamic public var buildingHighlightColor: UIColor = .defaultBuildingHighlightColor
     
+    @objc dynamic public var reducedAccuracyActivatedMode: Bool = false {
+        didSet {
+            self.userCourseView = reducedAccuracyActivatedMode ? UserHaloCourseView(frame: CGRect(origin: .zero, size: 75.0)) : UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0))
+            //TODO: add the weak gps connection banner below
```

Comment:
#2693 will add the banner code to NavigationViewController at a spot where it’ll be convenient to set this property as well.

Reply:
I see. I'll rebase this branch and follow the #2693 to add the property setting value here.